tests/console/puppet.pm

```diff
@@ -31,10 +31,12 @@ echo 'certname = puppetslave.local' >> /etc/puppet/puppet.conf
 echo '[master]' >> /etc/puppet/puppet.conf
 echo 'certname = puppetmaster.local' >> /etc/puppet/puppet.conf
 echo 'dns_alt_names = puppetmaster.local' >> /etc/puppet/puppet.conf
-systemctl start puppetmaster
-systemctl start puppet
 EOF
     assert_script_run($_) foreach (split /\n/, $puppet_conf);
+    assert_script_run 'systemctl start puppetmaster';
+    # check puppetmaster got started before starting puppetslave
+    assert_script_run 'systemctl show -p ActiveState puppetmaster.service|grep ActiveState=active';
+    assert_script_run 'systemctl start puppet';
```

Comment:
Why do you not just keep this commands in the above array? Also please add spaces around the `|` which is a more common style

Reply:
assert_script_run $puppet_conf which contains assert_script_run, does it a good style here?
Check ActiveState is a important change and it doesn't  belong to $puppet_conf, I think.

Since Dumitru is gone, I can take over the role of maintainer from now.